lib/fluent/plugin/in_tail.rb

```diff
@@ -117,6 +114,29 @@ def configure_tag
       end
     end
 
+    def configure_encoding
+      unless @encoding
+        if @from_encoding
+          log.warn "'from_encoding' parameter must be specified with 'encoding' parameter."
+          log.warn "'from_encoding' is ignored"
```

Comment:
This log is confused.
If `from_encoding` must be specified with `encoding`, we should raise an ConfigError.

In addition, no need two line logs. 2nd line should be merged into 1st.